base/fixtures/hello_cargo/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "hello_cargo"
+version = "0.1.0"
+edition = "2018"
```

Comment:
Needs to set `2018` instead of current `2021` as some rust dsitrib packages are quite old.